xml_converter/src/attribute/profession_filter.cpp

```diff
@@ -8,9 +8,9 @@
 #include "../string_helper.hpp"
 
 using namespace std;
-
-ProfessionFilter parse_ProfessionFilter(rapidxml::xml_attribute<>* input, vector<XMLError*> *errors) {
-    ProfessionFilter profession_filter;
-    profession_filter.parse(input, errors);
-    return profession_filter;
-}
+// //  Functionality will change for parsing
```

Comment:
No nee to comment this out, just delete it. If we need access to what it used to be we can look back into the git history.